spikeinterface/sorters/runsorter.py

```diff
@@ -211,7 +220,7 @@ def run_command(self, command):
 
 def run_sorter_container(sorter_name, recording, mode, container_image, output_folder=None,
                          remove_existing_folder=True, delete_output_folder=False,
-                         verbose=False, raise_error=True, with_output=True, **sorter_params):
+                         verbose=False, raise_error=True, with_output=False, **sorter_params):
```

Comment:
@JuliaSprenger after discussing with Sam we think that this should be `True` by default, even if for Tridesclous (only) the sorter needs to be installed also locally

Reply:
Ok, I changed the default value.